etc/kayobe/ansible/growroot.yml

```diff
@@ -76,4 +76,3 @@
         pv: "{{ pvs.stdout | from_json }}"
         disk: "{{ pv.report[0].pv[0].pv_name }}"
       become: True
-      when: "'NOCHANGE' not in growpart.stdout"
```

Comment:
Could you comment it out and add a comment about why this has been removed, with a link to the bug report

Reply:
sure